meteor-app/imports/ui/service/service-item-search.stories.js

```diff
@@ -0,0 +1,78 @@
+import React, { Component } from 'react'
+import { storiesOf } from '@storybook/react'
+import { action } from '@storybook/addon-actions'
+import SearchBar from './service-item-search'
+import faker from 'faker'
+
+const data = [
+  { title: 'item1', isMajMin: false },
```

Comment:
I would prefer this test data to live in a separate file, so that it can also be used in other places. Also, we have some realistic data already, which we can use to make this more 'real'